src/enzo/ReadParameterFile.C

```diff
@@ -1739,94 +1724,23 @@ int ReadParameterFile(FILE *fptr, TopGridData &MetaData, float *Initialdt)
 
 
 #ifdef USE_GRACKLE
```

Comment:
I think this `ifdef` can be removed for similar reasons since there is `ifdef` code inside `GrackleReadParameters`. This should allow you to remove all grackle ifdefs from this file, which should make it a bit easier to read.